src/catalog/metastore.rs

```diff
@@ -140,21 +140,21 @@ impl Metastore {
     async fn build_table(
         &self,
         table: TableObject,
-        store_options: &HashMap<String, HashMap<String, String>>,
+        store_options: &HashMap<String, (String, HashMap<String, String>)>,
```

Comment:
There's 4 Strings in this type and it took me a while to figure out what each of them was. Maybe add a comment about this?

Reply:
Good point, will do.